plugins/modules/acme_certificate_renewal_info.py

```diff
@@ -104,16 +102,15 @@
 msg:
   description:
     - Information on the reason for renewal.
-    - Should be shown to the user, as in case of ARI triggered renewal it can contain important
-      information, for example on forced revocations for misissued certificates.
+    - Should be shown to the user, as in case of ARI triggered renewal it can contain important information, for example on
+      forced revocations for misissued certificates.
   type: str
   returned: success
   sample: The certificate does not exist.
 
 supports_ari:
   description:
-    - Whether ARI information was used to determine renewal. This can be used to determine whether to
-      specify O(community.crypto.acme_certificate#module:include_renewal_cert_id=when_ari_supported)
+    - Whether ARI information was used to determine renewal. This can be used to determine whether to specify O(community.crypto.acme_certificate#module:include_renewal_cert_id=when_ari_supported)
```

Comment:
```suggestion
    - Whether ARI information was used to determine renewal. This can be used to determine whether to specify
      O(community.crypto.acme_certificate#module:include_renewal_cert_id=when_ari_supported)
```